apps/src/common-styles.module.scss

```diff
@@ -34,7 +34,8 @@
 //  More info here: https://github.com/code-dot-org/code-dot-org/pull/50895
 .purpleHeader {
   height: style-constants.$workspace-headers-height;
-  background-color: color.$purple;
+  // THIS IS AN OLD PURPLE COLOR. DO NOT USE THIS COLOR FOR NEW STYLES. ONLY USE IT FOR JAVALAB HEADER.
+  background-color: #7665A0;
```

Comment:
Worth making a constant (ex. `color.$deprecated-purple`) to make this extra clear? Not required though, if this is the only usage of it and we want to avoid it being used elsewhere.

Reply:
Yeah, that's the only place in the repo we use this deprecated-purple, so I think it's better to leave it as is